core/primitives/src/lib.rs

```diff
@@ -5,6 +5,8 @@ extern crate jemallocator;
 #[global_allocator]
 static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
 
+pub use borsh;
```

Comment:
what is this for? Doesn't make sense to me as borsh is an external library.

Reply:
It needs by validator to deserialize Storage data.

Reply:
Why don't you import borsh in that crate?

Reply:
I can't understand what's wrong with adding borsh for public usage here. Could you please decide it with @mikhailOK? I'm okay with any solution.

Reply:
Re-exporting solves inconsistencies, so I think it is a good practice: https://github.com/near/near-sdk-rs/issues/142